README.md

```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
+ 
+4. Please verify that host can access `api.inabit.com`  server (whitelisted IPs and ports).
+
+5. Please be aware that for completion of the process, you will need interact with your inabit account owner.
+
+## Quick Setup
+
+Before you begin, ensure all prerequisites are met, and that you have the *login token* provided by our representatives.
+
+Then, run this in terminal (linux).
+
+```shell
+# clone this repository
+git clone https://github.com/In-a-bit/inabit-remote.git
+cd inabit-remote
+# create 2 needed directories
+mkdir refresh
+mkdir dat
+# save the login token in file.
+# in the next line, replace <login token> with the token you got.
+echo "<login token>" > refresh/r.dat
+touch dat/k.dat
+# copy the .env.example to .env
+cp ./apps/approver/.env.example .env
+```
+
+Now, Open `.env` file in your text editor, and change the following values:
+
+  * APPROVER_URL - set here the URL of your app. 
+  This URL is the URL that this approver app is hosted on. Means - it will get http requests on this URL, from `inabit.com` servers. **In case you don't have some logic, so for convince, there is an option to set here a predefined url - see [mock validation](#mock-validation) part in this document. 
+  
+  * SECRET - set here a long secret.
+
+  * ORGANIZATION_NAME - set here your organization name.
+
+  * INABIT_API_BASE_URL - should be: https://api.inabit.com/graphql
+
+  * APPROVER_CREATOR_EMAIL - set here your email.
+
+  
+Then save the file, and close the editor.
+
+After the file is ready, build the docker, and run it with the following commands : 
+
+```shell
+docker compose -f docker-compose.prod.yml build approver
+docker compose -f docker-compose.prod.yml up -d approver
+```
+
+Now the last step, you should get the the _pairing token_ from the docker logs.
+In order to do it, run the following:
+
+```shell
+docker logs inabit-remote-approver-1
+
+# then you should search for this log
+# and copy the Pairing code 
+{
+  level: 'info',
+  message: 'Pairing code: <COPY THIS CODE>',
+  metadata: { timestamp: '2024-02-21T08:54:17.560Z' },
+  timestamp: '2024-02-21T08:54:17.560Z'
+}
+```
+
+**Send this code to you owner.**.
+
+
+Ask the owner to open the inabit app on their smartphone. They should have already received a notification from the inabit app requesting approval for the new Approver.
+
+The owner should tap on the notification and enter the provided code when prompted.
+
+After this step is done, you are ready to go!
+
+The next step to do is to create your API Wallet.
+
+If you would like to get deep knowledge, you can read the rest of this document.
+
 
 ## Deployment
 
 From within the root directory `inabit/inabit-remote`:
 
  ### Set Login Token
  
- The login token is approver's token valid for 30 days,
- 
- allowing authentication with Inabit.
+ The login token is approver's token valid for 30 days, allowing authentication with Inabit.
```

Comment:
is an approver's token that is valid for 30 days

Reply:
done.